### ISSUE TITLE
Batch Requests Support

### ISSUE THREAD

**Issue by jrow2286** [gitbrent/SpRestLib#52]
Any plans to support batch requests [https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/make-batch-requests-with-the-rest-apis](url)?

I love this library and sometimes will need to loop through a list to make many updates and SharePoint online throttles me. 

Thanks! #

**Comment by gitbrent** [comment]
Hi @jrow2286 

Thanks for the feedback and the suggestion.

As only the recent API supports batching, I imagine just adding a flag like `batch: true` would be how this is enabled along with an array of objects.

**Comment by jrow2286** [comment]
Yeah, that makes sense to me. Thanks for considering this enhancement!
